src/pages/commits/InactiveCommits/index.tsx

```diff
@@ -193,7 +193,6 @@ const InactiveCommitLine = styled.div`
 `;
 
 const TooltipContainer = styled.div`
-  width: 300px;
```

Comment:
We actually apply a `width` to a lot of tooltips in the codebase as a workaround for old LeafyGreen problems. I think you'll be able to find a lot of them by searching for `StyledTooltip`. It might be worth seeing in this PR or in another ticket if we can now remove those `width`s!